src/assets/scripts/client/aircraft/AircraftModel.js

```diff
@@ -2016,20 +2017,28 @@ export default class AircraftModel {
         // SPEED
         this.updateSpeedPhysics();
 
+        const offsetGameTime = TimeKeeper.accumulatedDeltaTime / GameController.game_speedup();
+        // const nextHistoricalPosition = [
```

Comment:
this entire block may be changed under #760.  The below `FIXME` should be addressed then